packages/flame/lib/src/text.dart

```diff
@@ -8,22 +8,72 @@ import 'extensions/size.dart';
 import 'extensions/vector2.dart';
 import 'memory_cache.dart';
 
-/// A Text Config contains all typographical information required to render texts; i.e., font size and color, family, etc.
-///
-/// It does not hold information regarding the position of the text to be render neither the text itself (the string).
-/// To hold all those information, use the Text component.
-///
-/// It is used by [TextComponent].
-class TextConfig {
+abstract class TextRenderer<T extends BaseTextConfig> {
```

Comment:
Maybe add a short explanation regarding what the class is for?

Reply:
@spydon I have added a dartdoc here, could you take a look if it is ok?

Reply:
Looks good!